src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  // eslint-disable-next-line no-param-reassign
+  const isMarried = person.partnerName;
```

Comment:
The logic for determining if a person is married relies on `partnerName`. Consider using `person.isMarried` for a more reliable check.